docs/integrations/containers-orchestration/opentelemetry/activemq-opentelemetry.md

```diff
@@ -0,0 +1,304 @@
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ActiveMQ-OpenTelemetry/ActiveMQ-OTel-Collection-architecture.png' alt="ActiveMQ OTel Collection architecture" />
+
+This App has been tested with following ActiveMQ versions:
+  * 5.17.4
+
+## Log Types and Metrics
+
+The Sumo Logic App for ActiveMQ assumes:
+
+- ActiveMQ app supports the below log format in log4j2.properties file
+
+  * [Audit Logs](https://activemq.apache.org/audit-logging):  Every management action made through JMX or Web Console management interface is logged in audit log files.
+
+    `appender.auditlog.layout.pattern=%d | %-5p | %m | %t%n`
+
+  * ActiveMQ Logs:
+
+    `appender.logfile.layout.pattern=%d | %-5p | %m | %c | %t%n%throwable{full}`
+
+
+- For a list of metrics that are collected and used by the app, see [ActiveMQ Metrics](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-metrics/docs/target-systems/activemq.md).
+
+
+## Fields Creation in Sumo Logic for ActiveMQ
+
+Following are the [Fields](/docs/manage/fields/) which will be created as part of ActiveMQ App install if not already present.
+
+* `messaging.cluster.name`. User configured. Enter a name to uniquely identify your ActiveMQ cluster. This cluster name will be shown in the Sumo Logic dashboards.
+* `messaging.node.name`. Has value of `host name`.
+* `messaging.system`. Has fixed value of `activemq`.
+* `sumo.datasource`. Has fixed value of `activemq`.
+
+If Process Metrics are enabled it will also create [fields for JMX metrics](/docs/integrations/app-development/opentelemetry/jmx-opentelemetry/#fields-creation-in-sumo-logic-for-jmx)
+
+
+## Prerequisites
+
+#### Configure logging in ActiveMQ:
+
+  1. By default, ActiveMQ logs (`audit.log` and `activemq.log`) are stored in the directory called `${ACTIVEMQ_HOME}/data/activemq.log`. Make a note of this logs directory.
+
+  2. [Enable auditing](https://activemq.apache.org/audit-logging) if not enabled by default
+
+     `ACTIVEMQ_OPTS="$ACTIVEMQ_OPTS -Dorg.apache.activemq.audit=true"`
+
+  3. Ensure that below log formats are present in [logging configuration](https://activemq.apache.org/how-do-i-change-the-logging). Also make sure appropriate log level is set.
+
+      * [Audit Logs](https://activemq.apache.org/audit-logging):
+
+        `appender.auditlog.layout.pattern=%d | %-5p | %m | %t%n`
+
+      * ActiveMQ Logs:
+
+        `appender.logfile.layout.pattern=%d | %-5p | %m | %c | %t%n%throwable{full}`
+
+
+
+#### Configure metrics in ActiveMQ:
+
+  1. Follow the instructions in [JMX - OpenTelemetry's prerequisites section](/docs/integrations/app-development/opentelemetry/jmx-opentelemetry/#prerequisites) to download the [JMX Metric Gatherer](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-metrics/README.md) used by the [JMX Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver#details).
```

Comment:
```suggestion
  1. Follow the instructions in [JMX - OpenTelemetry's prerequisites section](/docs/integrations/app-development/opentelemetry/jmx-opentelemetry#prerequisites) to download the [JMX Metric Gatherer](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-metrics/README.md) used by the [JMX Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver#details).
```

Reply:
The first link is broken because the JMX article has not yet been published (PR #2734).

Reply:
yes both of these will go live at same time